resources/js/src/app/components/itemList/filter/ItemDropdown.js

```diff
@@ -0,0 +1,54 @@
+import url from '../../../mixins/url'
+
+Vue.component("item-dropdown", {
+    mixins: [url],
+
+    props: [
+        "template",
+        "facet"
+    ],
+
+    data: function () {
+        return {
+            isShowDropdown: false
+        }
+    },
+
+    created() {
+        this.$options.template = this.template || "#vue-item-dropdown";
+
+        window.addEventListener('click', this.onClick);
```

Comment:
Please check if the event listener can be set via Vue.js instead of pure JS (like it's done with `selected`).

Reply:
Using Vue.js @click, I can listen for the event when it is clicked only on the component. In this case I need to check for any click on the window, so that I know is it clicked inside the component or the outside (and then I show or hide the component respectively). 
I don't know how to set this kind of event differently, using Vue.js components

Reply:
What if you used the click event on the component to open it and the blur event to close it again once it loses focus?

Reply:
Tried it. Blur event doesn't fire.